Dockerfile.windows

```diff
@@ -208,6 +211,40 @@ RUN `
     } `
   } `
   `
+  Function Build-GoTestSum() { `
+    $GotestsumPath=Join-Path -Path $Env:GOPATH -ChildPath "src\gotest.tools\gotestsum"; `
+    $GotestsumBinPath=Join-Path -Path $GotestsumPath -ChildPath "gotestsum.exe"; `
+    Write-Host "INFO: Building gotestsum version $Env:GOTESTSUM_COMMIT in $Env:GOPATH" `
+    $optsForGet = @('"get"', '"-d"', '"gotest.tools/gotestsum"'); ` 
+    $logFileName = 'gotestsum.get.out';  `
+    &go $optsForGet; `
+    $savedExitCode = $LASTEXITCODE; `
+    if ($savedExitCode -ne 0) {  `
+      Throw '"Failed getting gotestsum sources..."'  `
+    }; `
+    Write-Host "INFO:     Sources obtained for gotestsum..."; `
+    Push-Location $GotestsumPath; `
+    $optsForCheckout = @('"checkout"', '"-q"', """$Env:GOTESTSUM_COMMIT"""); ` 
+    $logFileName = 'gotestsum.git.out';  `
+    &git $optsForCheckout; `
```

Comment:
See my previous review; could those options be moved inline here?

Reply:
I feel that the above is much cleaner and easier to read. I am open to changing it if there is a reason why we should inline the arguments here.

Reply:
Perhaps personally, but in order to understand what this line does, I have look for where `$optsForCheckout` is declared; so read up two lines higher, mentally "remove" the additional quoting that's needed there, and "reconstruct" that this says;

```console
git checkout -q "$GOTESTSUM_COMMIT"
```